home-application/src/main/java/com/softserveinc/ita/homeproject/application/mapper/IHomeMapperImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.softserveinc.ita.homeproject.application.mapper;
+
+import com.softserveinc.ita.homeproject.homeservice.dto.BaseDto;
+import com.softserveinc.ita.homeproject.model.BaseView;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.lang.reflect.Type;
+
+@Component
+@RequiredArgsConstructor
+public class IHomeMapperImpl implements IHomeMapper {
```

Comment:
Why do we need it? What if we just use HomeMapper directly?

Reply:
Done.